### PR TITLE
[W.I.P.] Patch flaky SEPT tests on Windows by checking for dimension size

### PR DESCRIPTION
**Pull request by IshanMi**
## Description
Currently, four tests for SingleEntityPhiTensor fail on Windows when the tensor data generated by the pytest fixture is of size (1, 1):

- test_ne_shapes
- test_compress
- test_partition
- test_partition_axis

## Affected Dependencies
No changed or affected dependencies.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

I generated 1 million random tensors using the same method as was done in the pytest file, and none of them failed the dimensions check.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: packages/syft/tests/syft/core/tensor/autodp/bug_catcher.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+for iteration in {1..100}
```
Comment (rasswanth-s) [inline packages/syft/tests/syft/core/tensor/autodp/bug_catcher.sh:1]: Q: Should we remove this?

File: packages/syft/tests/syft/core/tensor/autodp/single_entity_phi_test.py [hunk 2]
```diff
@@ -17,7 +17,12 @@
 # Global constants
 ishan = Entity(name="Ishan")
 traskmaster = Entity(name="Andrew")
-dims = np.random.randint(10) + 3  # Avoid size 0
+dims = int(max(3, np.random.randint(10) + 3))  # Avoid size 0 and 1
```
Comment (rasswanth-s) [inline packages/syft/tests/syft/core/tensor/autodp/single_entity_phi_test.py:20]: Strange as to why it is generating tensor of (1,1) at this point

Comment (rasswanth-s) [reply packages/syft/tests/syft/core/tensor/autodp/single_entity_phi_test.py:20]: Is there any dimensional compression @IshanMi , during SEPT initialization

Comment (rasswanth-s) [reply packages/syft/tests/syft/core/tensor/autodp/single_entity_phi_test.py:20]: Sometimes, the CI additional randomness, It happened many times, if we run locally  for zillion times it passes, but in CI it fails

File: packages/syft/tests/syft/core/tensor/autodp/bug_catcher.sh (hunk 1 above)
Comment (IshanMi) [reply packages/syft/tests/syft/core/tensor/autodp/bug_catcher.sh]: Great catch, I just deleted it :)

**Review by rasswanth-s — APPROVED** [review]

**Review by iamtrask — APPROVED** [review]
such a strange bug

**Pull request merged**